### ISSUE TITLE
What about dwl?

### ISSUE THREAD

**Issue by spavlovich001** [bakkeby/dwm-flexipatch#426]
https://codeberg.org/dwl/dwl

Are there plans to create dwl-flexipatch? :)

**Comment by bakkeby** [comment]
I do not have any plans for that no.

One thing is creating it, but the most time consuming part would be maintaining and supporting it.

**Comment by spavlovich001** [comment]
> One thing is creating it, but the most time consuming part would be maintaining and supporting it.

Agree. And yet wayland is becoming mainstream (slowly) and sooner or later we will have to move to it

**Comment by bakkeby** [comment]
I don't think X11 is going away in the foreseeable future.

And if you are into mainstream stuff then I suppose that you should just be using gnome anyway :)

**Comment by speediegq** [comment]
Wayland is very much half-baked at the moment. Despite that, look how many people have adopted it. Whatever organizations such as Red Hat, Freedesktop and GNOME push is essentially guaranteed to take over eventually. Don't underestimate the power these organizations have over the users. X11 isn't going away, but Xorg is likely going to.

**Comment by spavlovich001** [comment]
> Wayland is very much half-baked at the moment. Despite that, look how many people have adopted it. Whatever organizations such as Red Hat, Freedesktop and GNOME push is essentially guaranteed to take over eventually. Don't underestimate the power these organizations have over the users. X11 isn't going away, but Xorg is likely going to.

I completely agree. It's a matter of time before X11 will no longer be supported, including in drivers. This will not happen immediately, but this process is irreversible, unfortunately
